e2e/integrations/basic-filter.feature

```diff
@@ -28,6 +28,8 @@ Feature: Test to verify addition of basic filter step to integrations
     Then click on the integration save button
     And she defines integration name "One step integration"
     And click on the "Save as Draft" button
+    Then Camilla is presented with "One step integration" integration details
+    And Camilla clicks on the "Done" link
```

Comment:
@lvydra fyi I've opened issue about buttons not being `button` element on Summary page https://github.com/syndesisio/syndesis-ui/issues/884.

Reply:
The button fix is merged via https://github.com/syndesisio/syndesis-ui/pull/885. Please adapt your PR accordingly. Also the syndesis-qe instance is updated to reflect it.